satellite/metrics/counter.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package metrics
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/rpc/rpcstatus"
+	"storj.io/storj/satellite/metainfo"
+)
+
+// counter implements the metainfo loop observer interface for data science metrics collection.
+//
+// architecture: Observer
+type counter struct {
+	remoteDependentObjectCount int64
+	inlineObjectCount          int64
+	totalObjectCount           int64
```

Comment:
Since it holds counts you don't really need the `ObjectCount` suffix for everything.